### DISCUSSION TITLE
mvt的显卡调用

### DISCUSSION THREAD

**Discussion by iforgiveu** [hooke007/mpv_PlayKit#502] [Q&A 问答]
补帧vs里，不同于rife的vpy配置，mvt的vpy配置并没有gpu调用设置，k7sfunc.py里也没找到与mvt相关的gpu调用设置，这导致我使用mvt补帧时只有核显在跑，该做什么来调用其它显卡呢？

**Comment by hooke007** [comment]
mvt纯cpu方案。是你的播放器在用核显
